docs/Getting Started/Example Apps/index.md

```diff
@@ -12,4 +13,208 @@ The example apps implement soft buttons, template text and images, a main menu a
+
+### Troubleshooting
+
+Sometimes things don't always go as planned, and so this section exists. If your app compiles and does NOT show up on the HMI, there are a few things to check out.
+
+#### TCP
+
+1. Make sure that you have changed the IP in ```SdlService.java``` to match the machine running Sdl Core. Being on the same network is also important.
+2. If you are sure that the IP is correct and it is still not showing up, make sure the Build Flavor that is running is ```tcpDebug```.
+3. If the two above dont work, make sure there is no firewall blocking the incoming port ```12345``` on the machine or VM running SDL Core. In the same breath, make sure your firewall allows that outgoing port.
+4. There are different network configurations needed for different virtualization software (virtualbox, vmware, etc). Make sure yours is set up correctly. Or use [Manticore](https://smartdevicelink.com/resources/manticore/).
+
+#### Bluetooth
+
+1. Make sure the build flavor ```multi_sec_offDebug ``` is selected.
+2. Ensure your phone is properly paired with the TDK
+3. Make sure Bluetooth is turned on - on Both the TDK and your phone
+4. Make sure apps are enabled on the TDK (in settings)
+!@
+
+@![javaEE, javaSE]
+### Introduction
+
+In this guide we take you through the steps to get our sample project, Hello Sdl, running and connected to Sdl Core as well as showing up on the generic HMI.
+
+Make sure that you follow the steps in [Installation](Getting Started With JavaEE/Installation) and [Integration Basics](Getting Started With JavaEE/Integration Basics) sections to create a new JavaEE SDL project before continuing this section.
+
+!!! NOTE
+The [Hello Sdl JavaEE](https://github.com/smartdevicelink/sdl_java_suite/tree/master/hello_sdl_java_ee) project includes samples for `SdlService` and `Main` classes that can be copied to your project.
+!!!
+
+### Getting Started
+
+We assume that you have [SDL Core](https://github.com/smartdevicelink/sdl_core) (We recommend Ubuntu 16.04) and an [HMI](https://github.com/smartdevicelink/generic_hmi) set up prior to this point. Most people getting started with this tutorial will be using Sdl Core and our Generic HMI. If you don't want to set up a virtual machine for testing, we offer [Manticore](https://smartdevicelink.com/resources/manticore/), which is a free service that allows you to test your apps in the cloud.
+
+!!! NOTE
+Sdl Core and an HMI or Manticore are needed to run Hello Sdl and to ensure that it connects.
+!!!
+
+### Configuring Core 
+
+To let Sdl Core connect to your app, first you will have to know the IP address of the machine that is running the Hello Sdl app. If you don't know what it is, running ```ifconfig``` in the terminal will usually let you see it for the interface you are connected with to your network. 
+
+After getting the IP address, you will have to set App ID, App Websocket Endpoint, and App Nicknames in Sdl Core to let it know where your instance of Hello Sdl is running. 
```

Comment:
```suggestion
After getting the IP address, you will have to set the app ID, app websocket endpoint, and app nicknames in Core's policy table. This will let Core know the location of your app's instance. 
```